### PR TITLE
Wind speed interpolation

### PR DESCRIPTION
**Pull request by ebbekyhl**
Added the function from [here](https://github.com/OskarSidor/atlite/blob/CMIP6_wind_interpolation/atlite/wind.py) that calculates the hub wind speed based on either 10-m wind speeds and roughness or based on an interpolation of wind speeds at two heights. 

I also added the argument "model" to the "convert_wind" function which should be either "interpolation" or "extrapolation" such that both methods can be applied and compared.

Currently, this has been tested for the EC-Earth3, which leads to a slightly higher capacity factor with the model="interpolation" setting (see jupyter notebook).

### CODE REVIEW

**Comment by martavp** [comment]
Thanks, @ebbekyhl I'll accept this. I cannot see the jupyter notebook with the test on EC-Earth 3, can you share it?

File: atlite/wind.py [hunk 1]
```diff
@@ -65,22 +69,52 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
...
+            raise AssertionError("model has to be either 'with_roughness' or 'without_roughness'")
+
+    # Wind speed extrapolation with the option to NOT use surface roughness, using the wind speed measured at two heights: 
+    if model == "without_roughness": 
+        print("from_height",from_height)
+        print("from_height2",from_height2)
+        from_name = "wnd{h:0d}m".format(h=int(from_height))
+        from_name2 = "wnd{h:0d}m".format(h=int(from_height2))         # definition of second wind speed (for instance at 100m)
+
+        wnd_spd = ds[from_name] * (to_height / from_height)**((1 / np.log(from_height)) * np.log(ds[from_name2] / ds[from_name]))
```
Comment (martavp) [inline atlite/wind.py:92]: Shouldn't the  equation be:

> wnd_spd = ds[from_name] * (to_height / from_height)**((np.log(ds[from_name2] / ds[from_name])/(np.log(from_height2) / np.log(from_height)) )

do you agree @ebbekyhl ?

Comment (ebbekyhl) [reply atlite/wind.py:92]: Yes, you are right. The former formulation only works when "from_height2" = 100 m and "from_height" = 10 m.

I would agree with Oskar's formulation (https://github.com/OskarSidor/atlite/commit/bc8c6228f2ad142b9550661460527a3e429d7003#r132404361):

```
wnd_spd = ds[from_name] * (to_height / from_height)**
(
np.log(ds[from_name2] / ds[from_name]) 
/ 
np.log(from_height2 / from_height)
)
```

**Comment by ebbekyhl** [comment]
@martavp You can find the comparison for EC-Earth-3 in the bottom of this jupyter notebook: https://github.com/martavp/atlite/blob/cmip6/sandbox_ebbe/download_cmip_cutout_and_process.ipynb

**Pull request merged**